lib/searchjoy/track.rb

```diff
@@ -5,11 +5,23 @@ def search_with_track(term, options = {}, &block)
         block.call(body) if block
       end
 
-      if options[:track]
+      if options[:track] && options[:execute]
```

Comment:
Is this needed if we hook into the `execute` method instead? Passing `execute: true` will cause searches to be inserted twice.